docs/cli-kit/ui-kit/contributing.md

```diff
@@ -0,0 +1,241 @@
+# Contributing to UI Kit
```

Comment:
Should we list this page [here](https://github.com/Shopify/cli/blob/main/docs/README.md#cli-kit)?

Reply:
Good point I'll add it.